### PR TITLE
fix(tile-select): support for wrapping adornment - FE-4207

### PR DESCRIPTION
**Pull request by dawidzarzycki**
### Proposed behaviour
Structure in file has been changed.
Styles has been changed to adopt new structure.
No changes required by end user.

fixes: #4219

### Current behaviour
Lack of possibility to wrap buttonAdornment

### Checklist

- [x] Commits follow our style guide
- [ ] <del> Screenshots are included in the PR if useful
- [ ] <del> All themes are supported if required
- [x] Unit tests added or updated if required
- [ ] Cypress automation tests added or updated if required
- [ ] <del> Storybook added or updated if required
- [ ] <del> Typescript `d.ts` file added or updated if required
- [ ] <del> Carbon implementation and Design System documentation are congruent

### Testing instructions
testable in codesandbox
https://codesandbox.io/s/mystifying-sound-myusm

### CODE REVIEW

File: src/components/tile-select/tile-select.style.js [hunk 1]
```diff
@@ -115,25 +114,23 @@ const StyledTileSelectInput = styled(Input)`
 const StyledTitleContainer = styled.div`
   display: inline-flex;
   align-items: flex-end;
-  margin-bottom: 8px;
+  margin-bottom: 3px;
```
Comment (edleeks87) [inline src/components/tile-select/tile-select.style.js:117]: Why has this been changed form 8px to 3px?

**Review by edleeks87 — COMMENTED** [review]
Just the one question @dawidzarzycki everything else looks fine to me so happy to approve after it's addressed

File: src/components/tile-select/tile-select.style.js (hunk 1 above)
Comment (dawidzarzycki) [reply src/components/tile-select/tile-select.style.js:117]: @edleeks87 
 I changed a bit file structure and some css so to have the same "look" as it was before I had to change the margin. Before we had adornments set with position absolute so it didn't count in spacing

Comment (edleeks87) [reply src/components/tile-select/tile-select.style.js:117]: The "look" has changed anyway though as now the `padding: 24px` is being respected by the TileSelect content (see pic below). IMHO it should remain as 8px as the DS describes how we should implement values as factors of 8 but I'm happy to go with the general consensus here if any other reviewers disagree 

image

Comment (dawidzarzycki) [reply src/components/tile-select/tile-select.style.js:117]: @edleeks87 good spot! I have to double check my values in styles files

**Review by edleeks87 — DISMISSED** [review]

**Review by DipperTheDan — APPROVED** [review]

**Review by DipperTheDan — APPROVED** [review]

File: src/components/tile-select/tile-select.style.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ const StyledTitle = styled.h3`
 `;
 
 const StyledActionButton = styled(Button)`
-  visibility: ${({ checked }) => (checked ? "visible" : "hidden")};
+  visibility: ${({ checked }) => (checked ? "visible;" : "hidden;")};
```
Comment (edleeks87) [inline src/components/tile-select/tile-select.style.js]: extra `;` here

File: src/components/tile-select/tile-select.style.js [hunk 3]
```diff
@@ -29,6 +29,7 @@ const StyledAdornment = styled.div`
   position: static;
   z-index: 200;
   margin-right: 16px;
+  visibility: ${({ children }) => (children ? "visible;" : "hidden;")};
```
Comment (edleeks87) [inline src/components/tile-select/tile-select.style.js]: extra `;` here

File: src/components/tile-select/tile-select.style.js (hunk 2 above)
Comment (dawidzarzycki) [reply src/components/tile-select/tile-select.style.js]: it should be ok now

File: src/components/tile-select/tile-select.style.js (hunk 3 above)
Comment (dawidzarzycki) [reply src/components/tile-select/tile-select.style.js]: it should be ok now

**Review by edleeks87 — DISMISSED** [review]

**Review by DipperTheDan — DISMISSED** [review]

**Review by edleeks87 — DISMISSED** [review]

**Review by DipperTheDan — DISMISSED** [review]

**Review by DipperTheDan — APPROVED** [review]

**Review by edleeks87 — APPROVED** [review]

**Review by DipperTheDan — APPROVED** [review]

**Comment by edleeks87** [comment]
Closing as new designs have made this work no longer valid

**Pull request closed without merging**